src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs

```diff
@@ -209,6 +218,15 @@ public async Task<IActionResult> Create([FromBody, Required] BuildData build)
         {
             Data.Models.Build buildModel = build.ToDb();
             buildModel.DateProduced = DateTimeOffset.UtcNow;
+
+            if (buildModel.Incoherencies == null)
```

Comment:
I would wrap this in a try/catch, logging a warning on failure. The graph build should be reliable, but:
- There are possibilities of it failing, like if github or azdo have issues.
- It doesn't directly affect the correctness of a build upload.